source/screens/caseScreens/CaseSummary.js

```diff
@@ -599,6 +690,20 @@ const CaseSummary = (props) => {
                 </Card>
               );
             })}
+
+            {journals.journal && journals.journal.notes.note.length > 0 && (
+              <Card>
+                <Card.Body color="neutral">
+                  <Card.Title colorSchema="neutral">Anteckningar</Card.Title>
+                  {journals.journal.notes.note.map((note) => (
+                    <>
```

Comment:
Missing `key` property, we could probably replace `<>` with `<View key={`journal-${note.label}`}>` (seems to work when I tested, and `note.label` is sufficiently unique).